src/main/java/javaserver/Methods/Option.java

```diff
@@ -0,0 +1,5 @@
+package javaserver;
+
+public class Option extends Handler {
+
+}
```

Comment:
I would leave empty / unimplemented / untested code out of the commit / pull request and add them in if they are fully tested and implemented